app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
```

Comment:
Variable Names and Meaning:

Consider using more meaningful variable names than _ for improved readability throughout the code.